frontend/src/components/common/LabelledIcon.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+
+interface ILabelledIconProps {
+  icon: ReactElement;
+  labelText: string;
+}
+/**
+ * Component that displays a large icon on top of a label.
+ * @param {ILabelledIconProps} param0
+ */
+export const LabelledIcon = ({ icon, labelText }: ILabelledIconProps) => {
+  return (
+    <StyledLabelledIcon>
+      {icon}
+      <StyledLabel>{labelText}</StyledLabel>
+    </StyledLabelledIcon>
+  );
+};
+
+const StyledLabelledIcon = styled.div`
```

Comment:
Wonder if we should start separating these into their own files as discussed last week.

Reply:
see above.